pkgs/tools/filesystems/rmount/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, nmap, jq, cifs-utils, sshfs, fetchFromGitHub, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name    = "rmount-${version}";
+  version = "1.0.1";
+  rev = "v${version}";
```

Comment:
Please move this declaration directly into the `fetchFromGitHub` call instead of calling `inherit rev;`